saml2oidcservers.yml

```diff
@@ -1,9 +1,10 @@
 # file: saml2oidcservers.yml
-# 
+#
 ---
 
 - hosts: saml2oidc
   roles:
-    #- ansiblebit.oracle-java
+    #- { role: ansiblebit.oracle-java, tags: java }
+    #- { role: infopen.openjdk-jdk, tags: java }
```

Comment:
Does the infopen.openjdk-jdk role need to be commented out?

Reply:
The java role was always commented out, in order to not be run on every execution of the playbook.
If you think that we don't need this, I can remove the comment for ` infopen.openjdk-jdk` role.

Reply:
Depends on the behaviour of the infopen.openjdk-jdk role. Does it ensures that java is installed or it will try to upgrade java every time it is run? Ideally we want the "state: present" behaviour we get with the standard package/apt tasks

Reply:
yes, the default behaviour is "state:present"
https://github.com/infOpen/ansible-role-openjdk-jdk/blob/develop/tasks/install_debian.yml#L9
( The item in the task: https://github.com/infOpen/ansible-role-openjdk-jdk/blob/develop/vars/os_family/debian.yml)
So, i'm going to remove the comment from the role